Assets/MRTK/Tests/PlayModeTests/BaseCursorTests.cs

```diff
@@ -34,7 +34,7 @@ public class BaseCursorTests
         public IEnumerator SetUp()
         {
             PlayModeTestUtilities.Setup();
-            TestUtilities.PlayspaceToOriginLookingForward();
+            TestUtilities.PlayspaceToArbitraryPose();
```

Comment:
Nit: For tests that use the non-origin/looking forward pose, it might be helpful to have some comments somewhere in the test stating that (i.e. "hey these tests use an arbitrary playspace pose, you will have to use playspace transformations like in the tests below" in order to translate vectors and poses into actual locations that the tests exist in - or something else that maybe is better english and actually makes more sense).

I think the main thing here is that even having some comments here would help - I think that some folks may add tests in the future and if they could potentially lose time if they are wondering why the positions are way off, but having some comments/breadcrumbs could help alleviate those maybe).

Reply:
@wiwei agreed 100%, I'll try to find some strategic spots and clear wordage to help folk spot where they need to be aware of this and what to look for as samples.